### DISCUSSION TITLE
What is the `Framework API`?

### DISCUSSION THREAD

**Discussion by shiny** [oven-sh/bun#6397] [Q&A]
Could someone please provide an explanation of the 'Framework API' described in the documentation?

- What exactly is the Framework API, and what can we anticipate from it?
- When will it be available?

**Comment by yossydev** [comment]
I don't know the details either, It looks like something that makes changes to the js parser and bundle.They are tentatively calling it Kit.
ref: https://github.com/oven-sh/bun/pull/13215

**added**.

KIT was rename to bake https://github.com/oven-sh/bun/pull/14335
